src/App.js

```diff
@@ -0,0 +1,200 @@
+import React, { Component } from 'react'
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import styled from 'styled-components';
+import Menu from './UI/Menu/Menu.jsx';
+import Game from './UI/Game.jsx';
+import Leaderboard from './UI/Leaderboard.jsx';
+import './main.css';
+import backgroundLight from './assets/images/bg-light.png';
+import backgroundDark from './assets/images/bg-night.png';
+import mainTheme from './assets/audio/main theme chill.mp3';
+import Callback from './UI/Callback';
+import { connect } from 'react-redux';
+import { changeTheme, changeMusic, fetchUser, fetchLeaderboard } from './store/actions';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: null,
+      leaderboard: {
+        easy: [],
+        medium: [],
+        hard: [],
+      },
+    };
+
+    this.getAllData();
+
+    this.musicVolumeHandler = this.musicVolumeHandler.bind(this);
+    this.soundVolumeHandler = this.soundVolumeHandler.bind(this);
+    this.mainTheme = new Audio(mainTheme);
+    this.mainTheme.loop = true;
+  }
+
+  getAllData() {
+    const token = localStorage.getItem('token') || null;
+    if (token) {
+      this.props.fetchUser(token);
+    }
+    this.props.changeTheme(localStorage.getItem('nightmode') === "true" || false);
+    this.props.fetchLeaderboard("easy");
+    this.props.changeMusic(localStorage.getItem('music') === "true" || false);
+  }
+
+  musicVolumeHandler(volume) {
+    this.setState({musicVolume: volume});
+    this.mainTheme.volume = volume;
+  }
+
+  soundVolumeHandler(volume) {
+    this.setState({soundVolume: volume});
+    this.moveSound.volume = volume;
+    this.saveAllData({...this.state, soundVolume: volume});
+  }
+
+  render() {
+    if (this.props.music) {
+      this.mainTheme.play();
+    } else {
+      this.mainTheme.pause();
+    }
+    this.mainTheme.volume = this.props.musicVolume;
+    return (
+      <Main nightmode={this.props.nightmode}>
+        <BrowserRouter>
+          <Route path='/callback' component={Callback}/>
+          <Layout>
+            <Switch>
+              <Menu />
+            </Switch>
+            <Game userHandler={this.userHandler} userdata={this.state.user}/>
+            <Leaderboard />
+          </Layout>
+        </BrowserRouter>
+      </Main>
+    )
+  };
+}
+
+const Main = styled.div`
```

Comment:
It's better to store these styled elements in `Elements` file or folder with all others styled elements